.jenkins/sarus_run_test.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e
+set -x
+
+# Set up the compute node environment
+module load daint-gpu
+module add /project/d107/install/modulefiles/
+module load cray-python gcloud sarus
+gcloud auth configure-docker
+
+
+# Set up the working directing for the c12 test
+cd examples
+. /project/d107/install/venv/sn_1.0/bin/activate
+python write_rundir.py ../tests/pytest/config/default.yml "./c12_test"
+deactivate
+
+cd ${PWD}/c12_test
+cp ../../.jenkins/job_jenkins_sarus .
+cp ../../.jenkins/md5.txt .
```

Comment:
`../..` is outside this repository, relative to the directory this file is in. If it is meant to be run from a different directory, could you use some `sh` or `bash` command to get the location of this file, and then write your paths relative to that location?

Reply:
The test is being performed in examples/c12_test so ../.. just refers to the top directory of the repository.  Don't see how adding more complexity with bash/shell commands helps here